spec/controllers/state_file/questions/nj_sales_use_tax_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe StateFile::Questions::NjSalesUseTaxController do
+  let(:intake) { create :state_file_nj_intake }
+  before do
+    sign_in intake
+  end
+
+  describe "#edit" do
+    render_views
+    it 'succeeds' do
+      get :edit
+      expect(response).to be_successful
+    end
+  end
+
+  describe "#update" do
+    it_behaves_like :return_to_review_concern do
+      let(:form_params) do
+        {
+          state_file_nj_sales_use_tax_form: {
+            untaxed_out_of_state_purchases: "yes",
+            sales_use_tax_calculation_method: "automated"
+          }
+        }
+      end
```

Comment:
We cannot. The conditional logic is not the same as an if/else in the ruby code. It's controlled by the `question-with-follow-up` classes from Honeycrisp. Because honeycrisp is a JS library, it does not get executed during a rspec unit test. So the content in question will always be on the page.

Discussed this in slack with CFA a little while ago ([link](https://njcio.slack.com/archives/C07GVSC8X3P/p1727295165359129)) and we agreed that in order to test these kinds of conditional follow-ups, we need to use the end-to-end feature tests. The plan is for us to add NJ end-to-end tests once we have some of the specific test personas that the PMs are developing over the next couple weeks.